lib/shopify_api/resources/engagement.rb

```diff
@@ -0,0 +1,4 @@
+module ShopifyAPI
```

Comment:
Can we remove this file because we do not have a corresponding resource in shopify called enagement